conformance/client/google-java/src/main/kotlin/com/connectrpc/conformance/client/java/JavaInvoker.kt

```diff
@@ -49,14 +50,4 @@ class JavaInvoker(
     override fun bidiStreamClient(): BidiStreamClient<*, *> {
         return JavaBidiStreamClient(client)
     }
-
-    companion object {
-        fun serializationStrategy(codec: Codec): SerializationStrategy {
```

Comment:
I moved this helper function into the new `JavaHelpers`, which contains other helper functions now that there are more things to adapt from the generated code to the model used by `conformance/client`.